Task1/Program.cs

```diff
@@ -10,7 +10,556 @@ class Program
+                {
+                    Console.WriteLine("yOU ENTERED INCORRECt value!");
+                    continue;
+                }
+
+                Console.WriteLine("Enter second number");
+                if (!Int32.TryParse(Console.ReadLine(), out int secondNumber))
+                {
+                    Console.WriteLine("yOU ENTERED INCORRECt value!");
+                    continue;
+                }
+
+                Console.WriteLine("What operation do you want to do? /n If you want multiply enter * /n If you want add enter + /n If you want devide enter / /n If you want subtract enter - /n");
+                string choice = Console.ReadLine();
+
+                switch (choice)
+                {
+                    case "+":
+                        Console.WriteLine(firstNumber + secondNumber);
+                        break;
+                    case "-":
+                        Console.WriteLine(firstNumber - secondNumber);
+                        break;
+                    case "*":
+                        Console.WriteLine(firstNumber * secondNumber);
+                        break;
+                    case "/":
+                        if (secondNumber == 0)
+                            Console.WriteLine("You can't devide to 0!!!");
+                        else
+                            Console.WriteLine(firstNumber / secondNumber);
+                        break;
+                    default:
+                        Console.WriteLine("Entered incorrect operation!");
+                        continue;
+                }
+                Console.WriteLine("If you want break enter E");
+                string choiseExit = Console.ReadLine();
+                if (choiseExit == "E")
+                {
+                    break;
+                }
+
+            }
+        }
+
+        public static void BetweenAB()
+        {
+            Console.WriteLine("Enter number A:");
+            int a = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Enter number B, B > A:");
+            int b = Convert.ToInt32(Console.ReadLine());
+            int sum = 0;
+            if (a < b)
+            {
+                for (int i = ++a; i < b; i++)
+                {
+                    sum += i;
+                }
+                Console.WriteLine($"The sum of numbers between A and B is: {sum}");
+
+                for (int i = ++a; i < b; i++)
+                {
+                    if (i % 2 != 0)
+                    {
+                        Console.WriteLine($"Odds: {i}");
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine("B shoud be more than A!");
+            }
         }
+
+        public static void NaturalNumber()
+        {
+            Console.WriteLine("Enter naturale number:");
+            int number = Convert.ToInt32(Console.ReadLine());
```

Comment:
Lets continue using TryParse() as it allows us knowing when parsing was not succeeded.